src/Evolution/Systems/Ccz4/Ricci.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Ccz4 {
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the spatial ricci tensor
+ *
+ * \details Computes the spatial ricci tensor as:
+ * \f{align}
```

Comment:
It's best to have a blank line before `\f{align}` and after `\f}` so that clang-format doesn't combine lines and break math rendering.